### ISSUE TITLE
describe for a unique secondary index returns inaccurate create index stmt

### ISSUE THREAD

**Issue by kmuthukk** [YugaByte/yugabyte-db#332]
The DESCRIBE operation on the UNIQUE index doesn't accurately report the CREATE statement corresponding to the index. In particular, it drops the UNIQUE tag.

It reports for example:
```CREATE INDEX ssn_index ON k.emp (ssn);```
instead of:
```CREATE UNIQUE INDEX ssn_index ON k.emp (ssn);```

Detailed example:
```
06:50 $ ./bin/cqlsh
Connected to local cluster at 127.0.0.1:9042.
[cqlsh 5.0.1 | Cassandra 3.9-SNAPSHOT | CQL spec 3.4.2 | Native protocol v4]
Use HELP for help.
cqlsh> CREATE KEYSPACE IF NOT EXISTS k;
cqlsh> USE k;
cqlsh:k> drop table emp;
cqlsh:k> CREATE TABLE emp(eid int PRIMARY KEY, ename TEXT, ssn TEXT)
     ...              WITH TRANSACTIONS = { 'enabled' : true };

cqlsh:k>
cqlsh:k> CREATE UNIQUE INDEX ssn_index ON emp (ssn) COVERING (ename);
cqlsh:k> DESCRIBE ssn_index;

CREATE INDEX ssn_index ON k.emp (ssn);
```

**Comment by robertpang** [comment]
Some changes are required in the Python driver that cqlsh relies on to retrieve the additional index metadata to describe.

**Comment by robertpang** [comment]
Fixed in cqlsh commit [8f2b5af](https://github.com/YugaByte/cqlsh/commit/8f2b5aff31175879899685f8b26d9da8733fd377), release  [v3.10-yb-3](https://github.com/YugaByte/cqlsh/releases/tag/v3.10-yb-3).
